mapie/estimators.py

```diff
@@ -503,7 +481,7 @@ def predict(self, X: ArrayLike) -> np.ndarray:
         n_alpha = len(alpha)
         y_pred = self.single_estimator_.predict(X)
         # At this point, y_pred is of shape (n_samples_test,)
-        if self.method in ["naive", "base"]:
+        if self.method in ["naive", "base"] or self.cv == "prefit":
```

Comment:
Du coup `cv = prefit` revient à utiliser la méthode naive ?

Reply:
Le point commun entre `naive` et `prefit`, c'est qu'il n'y a qu'un seul estimateur. La différence, c'est que `naive` calcul les résidus sur le jeu d'entraînement, alors que ` prefit` le fait sur un jeu de validation. Si l'on confond training et validation, alors c'est bien équivalent (c'est même un test unitaire)

Reply:
OK je comprends mieux. Je pense que ça mériterait d'ajouter une petite explication dans la docstring comme c'est d'ailleurs fait dans `CalibratedClassifierCV` qui mentionne explicitement "The user has to take care manually that data for model fitting and calibration are disjoint."

Reply:
Done